bindings/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,113 @@
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(VERSION 3.12)
+
+project(PortAudioCpp LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
```

Comment:
This is not required. I can successfully build without. As a general purpose library the requirements should be as low as possible.